### PR TITLE
Use importlib.resources to access spinegeneric/flags/*.png

### PR DESCRIPTION
**Pull request by kousu**
(this drops py 3.5 and 3.6 in favour of The Way Of The Future (i.e. the way of three years ago))

Depends on #172 being merged first.

### CODE REVIEW

**Comment by kousu** [comment]
@PaulBautin / @jcohenadad please review when you get

File: spinegeneric/cli/generate_figure.py [hunk 1]
```diff
@@ -280,8 +281,8 @@ def _get_flag(name):
         Get the flag of a country from the folder flags.
         :param name Name of the country
         """
-        path_flag = os.path.join(sg.__path__[0], 'flags', '{}.png'.format(name))
-        return plt.imread(path_flag)
+        with importlib.resources.path(spinegeneric.flags, f'{name}.png')) as path_flags:
```
Comment (valosekj) [inline spinegeneric/cli/generate_figure.py:284]:
```suggestion
        with importlib.resources.path(spinegeneric.flags, f'{name}.png') as path_flags:
```

There is typo in doubled parentheses.

**Comment by jcohenadad** [comment]
@kousu CI does not launch on your PRs:
image

maybe related to the fact that you opened a branch outside this repos, and it is protected? https://github.community/t/expected-waiting-for-status-to-be-reported/16727

File: spinegeneric/cli/generate_figure.py (hunk 1 above)
Comment (jcohenadad) [reply spinegeneric/cli/generate_figure.py:284]: thanks @valosekj, it is fixed in 0c69a46937ae04fc2697cf0e39b544196f2aadf4

**Pull request merged**